README.md

```diff
@@ -1,89 +1,74 @@
-## DeepDetect : Open Source Deep Learning Server & API
+<p align="center"><img src="https://www.deepdetect.com/img/icons/menu/sidebar/deepdetect.svg" alt="DeepDetect Logo" width="60%" /></p>
 
-[![Join the chat at https://gitter.im/beniz/deepdetect](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/beniz/deepdetect?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Build Status](https://travis-ci.org/beniz/deepdetect.png)](https://travis-ci.org/jolibrain/deepdetect)
+<h1 align="center"> Open Source Deep Learning Server & API</h1>
+
```

Comment:
This repo for instance https://github.com/PyTorchLightning/pytorch-lightning#continuous-integration has badging (ci:complete_passing), do we need a single badge like that (the full table would be overkill!)

Reply:
I'm not sure we need badge or table, but we need a section with the exact setup we recommended/support/test/validate/we-are-sure-at-100%-this-combination-works.

I will add it into a separate PR.

Reply:
Also, badges that aren't green 100% of the time are bad ideas.